deployment/systemd/util/install.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -e
+
+HERE="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+mkdir -p /etc/semaphore
+cp ${HERE}/../semaphore.service /etc/systemd/system
```

Comment:
need to call `systemctl daemon-reload` after adding/modifying the systemd unit file.